adhoc-build.sh

```diff
@@ -12,8 +12,8 @@ NAME="DuckDuckGo-$SUFFIX`date "+%Y-%m-%d-%H-%M"`"
 echo Building $NAME
 echo
 
-xcodebuild -scheme DuckDuckGo clean archive -configuration release -sdk iphoneos -archivePath $1/$NAME.xcarchive
-xcodebuild -exportArchive -archivePath $1/$NAME.xcarchive -exportOptionsPlist ./adhocExportOptions.plist -exportPath $1/$NAME
+xcodebuild -allowProvisioningUpdates -scheme DuckDuckGo clean archive -configuration release -sdk iphoneos -archivePath $1/$NAME.xcarchive
```

Comment:
You can add new devices to the adhoc profile and let Xcode download it from this script rather than using Xcode itself.

Reply:
Nice!